Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -225,7 +225,7 @@ public void onRequestPermissionsResult(int requestCode, //requestCode is what yo
 }
 
 ```
-Permissions should be explicitly requested for every needed permission, even if a similar permission from the same group may change in the future. Also permissions may be granted without user approval automatically.
+Permissions should be explicitly requested for every needed permission, even if a similar permission from the same group has already been requested. For applications targeting SDK 25 (Android 7.1) and older, Android will automatically give an application all the permissions from a permission group, if the user grants one of the requested permissions of that group. Starting with SDK 26 (Android 8.0), permissions will still automatically be granted if a user has already granted a permission from the same permission group, but the application still needs to explicitly request the permission. In this case, the `onRequestPermissionsResult` handler will automatically be triggered without any user interaction.
```

Comment:
Let's use this format with API Level instead of SDK and put the SDK into brackets:

Android 9 (API level 28)

Reply:
Done.